src/scrollend.js

```diff
@@ -1,4 +1,4 @@
-const supported = "onscrollend" in window
+const supported = typeof window == 'undefined' ? true : "onscrollend" in window;
```

Comment:
would it make sense to still do one last check instead of falling back to always `true`?

```suggestion
const supported = typeof window == 'undefined' ? typeof onscrollend != 'undefined' : "onscrollend" in window;
```

Reply:
No, don't think so. It'll just cause the rest of the logic to be skipped, which is fine.

Reply:
or maybe just don't bother with `window` or [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis#browser_compatibility) or `document`, and just do this?

```suggestion
const supported = typeof onscrollend != 'undefined';
```

Reply:
Doing `typeof window` is the only way in js to check if you have a `window` global object without raising an exception.
It's also the standard method in most projects to simply check whether you're in browser-land or not.

Reply:
what about

```suggestion
const supported = typeof window == 'undefined' && "onscrollend" in window;
```

Reply:
No, heavens (: that will explode in the server again 🙈